### PR TITLE
add: `tabby-deb`

### PR DESCRIPTION
**Pull request by AloneER0**
image

### CODE REVIEW

**Review by Henryws — CHANGES_REQUESTED** [review]
Add `packagelist` entry

**Comment by AloneER0** [comment]
Done

File: packages/tabby-deb/tabby-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name="tabby-deb"
+gives="tabby"
+breaks="${gives} ${gives}-app ${gives}-debian-patched-deb"
+version="1.0.187"
+description="A terminal for the modern age"
+hash="4ee191bad7c099b2952e764d4342002fc2c93d48433ebae6a8cc110c5c1d1de7"
+maintainer="AloneER0"
```
Comment (Henryws) [inline packages/tabby-deb/tabby-bin.pacscript:7]: Add an email.

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — CHANGES_REQUESTED** [review]
Rename the file from `tabby-bin.pacscript` to `tabby-deb.pacscript`

**Comment by AloneER0** [comment]
Should be ok now

File: packages/tabby-deb/tabby-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name="tabby-deb"
+gives="tabby"
```
Comment (Henryws) [inline packages/tabby-deb/tabby-deb.pacscript:2]:
```suggestion
gives="tabby-terminal"
```

File: packages/tabby-deb/tabby-deb.pacscript [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+breaks="${gives} ${gives}-app ${gives}-debian-patched-deb"
```
Comment (Henryws) [inline packages/tabby-deb/tabby-deb.pacscript:3]:
```suggestion
breaks="tabby tabby-app"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**